client/components/ProjectItem.tsx

```diff
@@ -25,15 +25,15 @@ export default function ProjectItem({
             </button>
             {dropdown && (
                 <div className="projectContent">
-                    <div>
+                    <div className="apiDiv">
                         <b>API Key:</b> {project.apiKey}
```

Comment:
Can you display `project_id` here? When discussing with Flora it sounded like you had done this but it's not here (unless it's in a different PR). Either way it needs to be visible or no one can setup the logger to use the app.

Reply:
Yup I will do it in mp/main-responsive

Reply:
Yep it's all good there.